.env.example

```
@@ -30,6 +30,7 @@ KBIN_META_DESCRIPTION="content aggregator, content voting, discussion and micro-
 KBIN_META_KEYWORDS="mbin, content aggregator, open source, fediverse"
 KBIN_HEADER_LOGO=false
 KBIN_FEDERATION_PAGE_ENABLED=true
+MBIN_SIDEBAR_SECTIONS_LOCAL_ONLY=true
```

```suggestion
MBIN_SIDEBAR_SECTIONS_LOCAL_ONLY=false
```

matching the docker example config, I think we should have this default to all as it seemed like there was more opinions of preferring it all rather than local at time of #141 though if there's strong opinions we could always recheck